generic_expression_patterns_modules/GSEA_analysis.R

```diff
@@ -0,0 +1,52 @@
+## Run this once to setup environment
+## Used R 3.6.3
+#if (!requireNamespace("BiocManager", quietly = TRUE))
+#  install.packages("BiocManager")
+
+#BiocManager::install("clusterProfiler")
+
+#library(clusterProfiler)
+
+find_enriched_pathways <- function(DE_stats_file,
+                                  pathway_DB,
+								  statistic){
```

Comment:
I'm not sure how this happens. Unfortunately since its in R, the auto-formatting I have setup isn't working here. I'm still new to using these auto-formatters so I'm not sure if there is one that people normally use for R -- didn't find one in a quick google search.